addon/components/basic-dropdown.ts

```diff
@@ -279,28 +277,32 @@ export default class BasicDropdown extends Component<Args> {
   }
 
   _getDestinationId(): string {
-    let config = getOwner(this).resolveRegistration('config:environment');
-    let id: string;
-    if (config.environment === 'test' && (typeof FastBoot === 'undefined')) {
-      if (DEBUG) {
-        if (requirejs.has('@ember/test-helpers/dom/get-root-element')) {
-          try {
-            return requirejs('@ember/test-helpers/dom/get-root-element').default().id as string;
-          } catch(ex) {
-            // no op
-          }
-        }
-        let rootView = document.querySelector('#ember-testing > .ember-view');
+    let config = getOwner(this).resolveRegistration("config:environment");
+
+    // This takes care of stripping this code out if not running tests
+    if (macroCondition(isTesting())) {
```

Comment:
Does this work both when running `ember test` and also when running the dev server but visiting the `/tests` route?

Reply:
Yeah, that is actually quite cool, it will even work when e.g. running `ember test --environment=production`.